pkgs/by-name/at/atuin-desktop/package.nix

```diff
@@ -68,6 +69,9 @@ rustPlatform.buildRustPackage (finalAttrs: {
     tauriBuildFlags+=(
       "--config"
       "$tauriConfPath"
+      # Skips the version mismatch check (and accepts the consequences)
+      # ref: https://github.com/atuinsh/desktop/issues/313
+      "--ignore-version-mismatches"
```

Comment:
Assuming this program has a `--version` CLI flag, we could get some additional confidence by using [versionCheckHook](https://nixos.org/manual/nixpkgs/unstable/#versioncheckhook), which is a good sanity check that a program runs and prints the expected version. It doesn't help us test the GUI works, of course.

Feel free to ping me in review if you decide to add it.